### ISSUE TITLE
Export session participants

### ISSUE THREAD

**Issue by rbento1096** [uatisdeproblem/egm-app#128]
As an admin I'd like to be able to download the details about the participants for each session so I can share it with the speakers.

This data is saved on the registrations table and the most useful one is the `name` of the participant his esn country (if existent)

For this feature I'm thinking of 2 ways this can be processed:
- as a single call to the back-end that will scan all session registrations (at this time they are 5K+) and generate an excel with all data
- Add a button to the front-end that works as the current excel export of the users page (we can query the registrations database by `sessionId`to get all users registered in a session). This would be simpler but would require more manual work from the organizers. Excel should have a name = session.code or session.name if code is not defined.

Ideally we could have both.

**Comment by uatisdeproblem** [comment]
I implemented both the proposed scenarios.

The feature is online for testing in the development environment. Note: since there aren't many registrations in the dev environment, I tested by registering my user to _"How I Enforced my Inclusion in an Unfriendly Environment"_ (random session). It will also need to be tested with higher volumes of data in the production environment.

A few notes:
- Since the session data is repeated for each registration row in the spreadsheet, I added just the main data; feel free to add any more columns you like. 
   <img width="663" alt="Screenshot 2024-03-23 at 14 23 39" src="https://github.com/uatisdeproblem/egm-app/assets/3777036/25deed9c-7c82-4eaa-b7fd-36cb248e04a5">
- For sessions that don't have any registration, I added a row anyway (with empty data for the participants' columns); if you prefer not to include the sessions without registrations in the spreadsheet, we can change it.
- The export is available only to users with "canManageContents" permissions.

The feature is available in three areas of the app:

1. The Manage menu (download ALL sessions registrations):
![Screenshot 2024-03-23 at 14 13 58](https://github.com/uatisdeproblem/egm-app/assets/3777036/31e2c854-b317-4017-944c-49f17f498535)

2. The sessions list, with a "current session" selected:
![Screenshot 2024-03-23 at 14 13 07](https://github.com/uatisdeproblem/egm-app/assets/3777036/dc8b355f-b77c-4053-ae3d-641d12cf6cb7)

3. The detail of a session:
![Screenshot 2024-03-23 at 14 13 35](https://github.com/uatisdeproblem/egm-app/assets/3777036/368948ef-bb35-4525-8dc9-4bdef4a068ec)

I will now open the PR.
